src/store/traction/ont/pools/actions.js

```diff
@@ -289,18 +289,18 @@ export default {
    * @param rootState the vuex rootState object. Provides access to the current state
    * @param commit the vuex commit object. Provides access to mutations
    */
-  fetchOntRequests: async ({ commit, rootState }, filter) => {
+  fetchOntRequests: async ({ commit, rootState }, filter, page) => {
     const request = rootState.api.traction.ont.requests
-    const promise = request.get({ filter: filter })
+    const promise = request.get({ page, filter })
     const response = await handleResponse(promise)
 
-    const { success, data: { data } = {}, errors = [] } = response
+    const { success, data: { data, meta = {} } = {}, errors = [] } = response
```

Comment:
Again we are making lots of the same changes across the board. Seems like it points to a code smell. Should we use a shared method or a type?

Reply:
Yeah agreed. Should be a separate story I think as all API calls could be refactored like this.